pandas-stubs/core/indexes/datetimes.pyi

```diff
@@ -87,23 +90,24 @@ def date_range(
     start: str | DatetimeLike | None = ...,
     end: str | DatetimeLike | None = ...,
     periods: int | None = ...,
+    # TODO: Test timedelta and Timedelta, update pandas docs
     freq: str | timedelta | Timedelta | BaseOffset = ...,
     tz: str | tzinfo = ...,
     normalize: bool = ...,
-    name: str | None = ...,
-    inclusive: IntervalClosedType = ...,
-    **kwargs,
+    name: Hashable | None = ...,
+    inclusive: Literal["left", "right"] | None = ...,
```

Comment:
Based on the docs, this change is incorrect.  `IntervalClosedType` is the right type here.